server/src/migrations/1761765111646_add-user-trials.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @type {import('node-pg-migrate').ColumnDefinitions | undefined}
+ */
+export const shorthands = undefined
+
+/**
+ * @param pgm {import('node-pg-migrate').MigrationBuilder}
+ * @param run {() => void | undefined}
+ * @returns {Promise<void> | void}
+ */
+export const up = pgm => {
+  pgm.createTable('user_trials', {
+    user_id: { type: 'text', notNull: true, unique: true },
+    trial_start_at: { type: 'timestamptz' },
```

Comment:
Do we want timestamp with time zone or something more standard across timezones?

Reply:
I don't think so. Timezone isn't something that should be stored at the data level. Instead I think we're good with just storing a Unix timestamp and we can convert it in the app if we ever need to show a local DateTime.

Reply:
timestamptz includes timezone I think

Reply:
resolved offline -- using timezones to stay consistent with the rest of our repos